### PR TITLE
[WIP] - SSUI beginnings

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 52d4406c4d71cc3d847907fd7d315d773731a055:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
Nice. But - how we split the navigation tree? I believe `ui_navigate`'s navigation tree is a 'static' variable in the `ui_navigate` module itself. Or will we do it differently?

**Comment by psav** [comment]
Yes that's the next to tackle. I'm perhaps thinking two trees, that get used depending on the ipapp mode

**Comment by psav** [comment]
Need to think about this.......long.....and hard

**Comment by mfalesni** [comment]
@psav That sounds like juggling with other module's variables ...

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 404e9fa8d24213c6c23b6c9da6175a295ce07def:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e9f445e9ae92c2c759ede5d866be03bb03e06ac5:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/ssui/conftest.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def revert_to_default():
+    store.current_appliance.mode = "default"
+    browser.quit()
+
+
+@pytest.fixture(scope="module", autouse=True)
+def set_ssui_mode(request):
+    store.current_appliance.mode = "ssui"
```
Comment (mfalesni) [inline cfme/tests/ssui/conftest.py]: shouldn't we branch the nav tree right at its root than having modes? Just curious ...

**Comment by dajoRH** [comment]
Lint report for commit 919e28c5a4e84b7869bac98259362fa3524dd7c4:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/ssui/conftest.py (hunk 1 above)
Comment (psav) [inline cfme/tests/ssui/conftest.py]: As you can see below, I started adding in the menu nodes. The issue is that force_navigate is very tied into the CFME main interface. I wonder if this is just going to have to wait until FW3.0, but there is a part of me that is thinking that we will have multiple Endpoint classes, with their own navigation menus, and their own force_navigates. Perhaps I will try knocking something up on Monday for this.

Comment (mfalesni) [inline cfme/tests/ssui/conftest.py]: Yes, I can see that now.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
Will almost certainly never get merged as fw3 will address

**Pull request closed without merging**